Source/WebCore/platform/mac/ScrollingEffectsController.mm

```diff
@@ -635,6 +640,15 @@ static inline WheelEventStatus toWheelEventStatus(PlatformWheelEventPhase phase,
     startDeferringWheelEventTestCompletion(WheelEventTestMonitor::DeferReason::ScrollSnapInProgress);
 }
 
+void ScrollingEffectsController::scheduleScrollendTimer()
+{
+    static const Seconds discreteScrollDelay = 100_ms;
+    m_discreteScrollendTimer = m_client.createTimer([this] {
```

Comment:
This is creating a new timer on every event. Can't we reschedule instead, which is more efficient.

Reply:
Fixed.